build/amis/ansible/roles/common/defaults/main.yml

```diff
@@ -35,9 +34,9 @@ common_debs:
 - jq
 - nfs-client
 
-common_pinned_debs:
-- "cloud-init={{ cloud_init_version }}"
 common_extra_debs: []
 common_redhat_epel_rpm: "https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm"
-common_redhat_cloud_init_copr_rpm: "https://copr.fedorainfracloud.org/coprs/jdetiber/cloud-init/repo/epel-7/jdetiber-cloud-init-epel-7.repo"
-common_deb_cloud_init_ppr_repo: "ppa:detiber/cloud-init"
\ No newline at end of file
+common_redhat_epel_rpm_checksum: "sha256:33b8b8250960874014d7c66d04ef12df3a71277b9eb6f2146bba7553daeaf910"
+common_cloud_init_kubeadm_module: "https://raw.githubusercontent.com/detiber/cloud-init/44f9570f3005ace405c4899aded97d51873c7664/cloudinit/config/cc_kubeadm.py"
```

Comment:
Why not just add the file to `build/roles/common/files/`? That would remove the need for the remote call.

Reply:
That's what I did initially, but it's also potentially GPL licensed code, so wasn't sure how to put it in the repo.

Reply:
I'm good with re-licensing to Apache, would just need approval from Craig.

Reply:
oh, didn't read fully. It's dual licensed. I'll move it over.